docs/source/app_developers_guide/docker.rst

```diff
@@ -2,696 +2,672 @@
 Using Sawtooth with Docker
 **************************
 
+This tutorial explains how to set up Hyperledger Sawtooth for application
+development using a multi-container Docker environment. This step-by-step
+procedure shows you how to start Sawtooth and connect to the necessary Docker
+containers, then walks you through the following tasks:
```

Comment:
"This tutorial" followed by "This step-by-step procedure" feels awkward. Can the second sentence just start with "It shows"?

Reply:
Fixed.